Makefile

```diff
@@ -299,15 +299,17 @@ live-docs:
 
 .PHONY: apix-ref-docs
 apix-ref-docs:
-	crd-ref-docs \
+	GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on go install -v github.com/elastic/crd-ref-docs@latest
```

Comment:
Can you follow the pattern we are using for the other tools ([xref](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/Makefile#L362-L419))?